Tensile/Tests/unit/test_Utilities.py

```diff
@@ -0,0 +1,74 @@
+################################################################################
+#
+# Copyright (C) 2020-2022 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+################################################################################
+
+from Tensile.Utilities.String import splitDelimitedString
+
+def test_splitDelimitedString():
+    # Test `all` architectures
+    archs = "all"
+    expected = {"all"}
+    result = splitDelimitedString(archs, {";", "_"})
+    assert result == expected, f"arch `{archs}` should parse to {expected} but instead maps to {result}"
+
+    # Test all architectures, but supplied separately with semicolon delimieter (cli usage)
```

Comment:
Are there missing archs here or is the comment out of sync?

Reply:
Out of sync, deleting comments, the tests are readable as is.